config/schema_config.yaml

```diff
@@ -15,6 +15,27 @@ position entity:
     start: int
     end: int
 
+anatomical entity:
+  is_a: biological entity
+  description: >-
+    A subcellular location, cell type or gross anatomical part
+  exact_mappings:
+    - UBERON:0001062
+    - WIKIDATA:Q4936952
+      # UMLS Semantic Group "Anatomy"
+    - UMLSSG:ANAT
+      # UMLS Semantic Type "Anatomical Structure"
+    - STY:T017
+    - FMA:62955  # anatomical entity
+    - CARO:0000000   # anatomical entity
+    - SIO:001262  # anatomical entity
+    - STY:T029  # Body Location or Region
+    - STY:T030  # Body Space or Junction
+  id_prefixes:
```

Comment:
What the purpose `id_prefixes`? Is it used in the writer class to deterimine the type of a node?